tests/Faker.Tests/Common/NameTests.cs

```diff
@@ -25,5 +25,19 @@ public void Should_Get_Suffix()
 
 			Assert.That(new[] { suffix }, Is.SubsetOf(possibleSuffixes));
 		}
+
+		[Test]
+		[Repeat(100000)]
+		public void Compute_Time_Elapsed_By_The_First_Method()
+		{
+			Name.First();
```

Comment:
This unit test has no value, if you need to test for elapsed time
you would need to wrap Name.First() inside a foreach loop instead of using Repeat,
then use the Stopwatch class (could be NUnit has something built in, haven't checked) to measure the time used, while testing that it
is under a certain time frame.